src/harness/scripts/migrate-0.9.x.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+# pass 2 file names to this script, it will yield a output
+#./migrate-0.9.x.sh -f <override.yaml>
+SCRIPT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+while getopts ":f:c:" opt; do
+  case ${opt} in
+    f )
+      overrideFile=$OPTARG
+      ;;
+    \? )
+      echo "Invalid option: $OPTARG" 1>&2
+      echo "use ./migrate-0.9.x.sh -f <override.yaml>  "
+      ;;
+    : )
+      echo "Invalid option: $OPTARG requires an argument" 1>&2
+      ;;
+  esac
+done
+shift $((OPTIND -1))
+
+if command -v yq &>/dev/null; then
+    echo "yq is installed."
+else
+    echo " yq is not installed."
+    echo "\n For linux system, Run the following: \n ------------------- \n wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq \n ------------------- "
+    echo "\n For Windows system, Run the following: \n ------------------- \n choco install yq \n ------------------- "
+    echo "\n For other system and information, visit : \n ------------------- \n https://github.com/mikefarah/yq \n ------------------- "
+fi
+old_file=$overrideFile
+new_file_name="newOverride.yaml"
```

Comment:
consider new file name to be based on input file name.

Reply:
done